README.md

```diff
@@ -0,0 +1,98 @@
+<a href="https://significa.co"><img src="https://user-images.githubusercontent.com/4838076/70076649-20d29b00-15f7-11ea-9379-e2fa1889a525.png" alt="logo" width="300px"></a>
+
+[![React][React.js]][React-url]
+
+A Zero dependency opinionated Significa's i18n package for React.
+
+
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#messages-object">Messages object</a></li>
+    <li><a href="#message-parameters">Message parameters</a></li>
+    <li><a href="#license">License</a></li>
+  </ol>
+</details>
+
+## Usage
+
+Include the provider on the top level of your app.
+
+```js
+import { I18nProvider } from '@significa/react-i18n'
+
+<I18nProvider messages={i18nMessages}>
+  <App />
+</I18nProvider>
+```
+
+You can either use the `useTranslation` helper or the `Translation` component
+
+```js
+import { useTranslation } from '@significa/react-i18n'
+
+const YourComponent = () => {
+  const { t } = useTranslation()
+
+  return (
+    <div>
+      <p>{t.yourMessage}</p>
+    </div>
+  )
+}
+```
+
+```js
+import { Translation } from '@significa/react-i18n'
+
+const YourComponent = () => {
+  return (
+    <div>
+      <Translation id="yourMessage" />
+    </div>
+  )
+}
+```
+
+## Messages object
+
+Your messages object should be according to the following example.
+
+```
+{
+  id: message,
+  anotherId: anotherMessage
+}
+```
+
+You have to deal the language switcher outside this package.
+If using Next.js you'll have that out of the box
```

Comment:
not sure what you mean by this.
maybe we should move this paragraph next to the Provider and explain that we should only load the current language's strings to the provider?

Reply:
@pdrbrnd I think it's more clear now, let me know what you think